app/assets/v2/js/grants/detail.js

```diff
@@ -176,36 +178,41 @@ $(document).ready(function() {
     let deployedSubscription = new web3.eth.Contract(compiledSubscription.abi, contract_address);
 
     web3.eth.getAccounts(function(err, accounts) {
-      deployedSubscription.methods.changeOwnership(
-        contract_owner_address
-      ).send({
-        from: accounts[0],
-        gasPrice: 8000000000
-      }).on('transactionHash', function(transactionHash) {
-        const linkURL = get_etherscan_url(transactionHash);
-
-        document.issueURL = linkURL;
-        $('#transaction_url').attr('href', linkURL);
-        $('.modal .close').trigger('click');
-        enableWaitState('#grants-details');
-      }).on('confirmation', function(confirmationNumber, receipt) {
-        $.ajax({
-          type: 'post',
-          url: '',
-          data: {
-            'contract_owner_address': contract_owner_address
-          },
-          success: function(json) {
-            window.location.reload(false);
-          },
-          error: function() {
-            _alert({ message: gettext('Changing the contract owner address failed to save. Please try again.') }, 'error');
-          }
+      deployedSubscription.methods
+        .changeOwnership(contract_owner_address)
+        .send({
+          from: accounts[0],
+          gasPrice: 8000000000
+        })
+        .on('transactionHash', function(transactionHash) {
+          const linkURL = get_etherscan_url(transactionHash);
+
+          document.issueURL = linkURL;
+          $('#transaction_url').attr('href', linkURL);
+          $('.modal .close').trigger('click');
+          enableWaitState('#grants-details');
+        })
+        .on('confirmation', function(confirmationNumber, receipt) {
+          $.ajax({
+            type: 'post',
+            url: '',
+            data: {
+              'contract_owner_address': contract_owner_address
+            },
+            success: function(json) {
+              window.location.reload(false);
+            },
+            error: function() {
+              _alert({ message: gettext('Changing the contract owner address failed to save. Please try again.') }, 'error');
+            }
+          });
```

Comment:
@PierrickGT could you undo the linting you've done here ? 
It makes it confusing for reviewers to see what exactly your changes are

Reply:
I've fixed it: https://github.com/gitcoinco/web/pull/4793/commits/3a57279e3300b250ac2a80e1bc236b9302c8b039